Source/iop/Iop_Dmac.cpp

```diff
@@ -19,11 +19,11 @@ using namespace Iop::Dmac;
 CDmac::CDmac(uint8* ram, CIntc& intc)
     : m_ram(ram)
     , m_intc(intc)
-    , m_channelSpu0(CH4_BASE, CHANNEL_SPU0, *this)
-    , m_channelSpu1(CH8_BASE, CHANNEL_SPU1, *this)
-    , m_channelDev9(CH9_BASE, CHANNEL_DEV9, *this)
-    , m_channelSio2In(CH11_BASE, CHANNEL_SIO2in, *this)
-    , m_channelSio2Out(CH12_BASE, CHANNEL_SIO2out, *this)
+    , m_channelSpu0(CH4_BASE, CHANNEL_SPU0, Iop::CIntc::LINE_DMA4, *this)
+    , m_channelSpu1(CH7_BASE, CHANNEL_SPU1, Iop::CIntc::LINE_DMA8, *this)

```

Comment:
While we're at it, I think we should rename `LINE_DMA4` and `LINE_DMA8` to `LINE_DMA_SPU0` and `LINE_DMA_SPU1`

Reply:
Done